addons/website/models/res_partner.py

```diff
@@ -28,15 +24,15 @@ def google_map_img(self, zoom=8, width=298, height=298):
             'sensor': 'false',
             'key': google_maps_api_key,
         }
-        return urlplus('//maps.googleapis.com/maps/api/staticmap', params)
+        return '//maps.googleapis.com/maps/api/staticmap?'+werkzeug.urls.url_encode(params)
 
     @api.multi
     def google_map_link(self, zoom=10):
         params = {
             'q': '%s, %s %s, %s' % (self.street or '', self.city or '', self.zip or '', self.country_id and self.country_id.name_get()[0][1] or ''),
             'z': zoom,
         }
-        return urlplus('https://maps.google.com/maps', params)
+        return 'https://maps.google.com/maps' + werkzeug.urls.url_encode(params)
```

Comment:
shouldn't it be `werkzeug.urls.url_encode_plus` instead?

Reply:
> shouldn't it be `werkzeug.urls.url_encode_plus` instead?

It doesn't exists AFAIK